remoteit/desktop
# Desk 1079 refactor event components

### Description
refactor event components


### Test plan



1. Sign in
2. Go to device
3. Click on logs
4. You should see working correctly

### Ticket

https://remoteit.atlassian.net/browse/DESK-1079
